src/delivery-options/config/exports/PickupExportValues.js

```diff
@@ -32,21 +32,25 @@ export class PickupExportValues extends ExportValues {
       return;
     }
 
+    const foundLocation = configBus.pickupLocations.find((data) => {
+      return data.location_code === pickupLocationName;
```

Comment:
Hier zou je hetzelfde kunnen doen als wat Alex voorstelde

Reply:
dan krijg je dus een eslint error ha... ik kijk wel een keer of die weg kan, dat het alleen geldt wanneer je zelf variabelen aanmaakt ofzo